src/features/packages/pack/nugetPackageBuilder.test.ts

```diff
@@ -0,0 +1,106 @@
+import { NuGetPackageBuilder } from "./nugetPackageBuilder";
+import fs, { rmdir } from "fs";
+import os from "os";
+import path from "path";
+import AdmZip from "adm-zip";
+import { Logger } from "../../../logger";
+
+describe("Can create a NuGet packages", () => {
+    const logger: Logger = {
+        debug: (m) => console.log(m),
+        info: (m) => console.log(m),

```

Comment:
No need for `warn`?

Reply:
Not for this test, the code only writes debug and info messages.